src/broadcast/broadcastmanager.cpp

```diff
@@ -6,14 +6,15 @@
 #include "engine/sidechain/enginesidechain.h"
 #include "soundio/soundmanager.h"
 
-BroadcastManager::BroadcastManager(UserSettingsPointer pConfig,
+BroadcastManager::BroadcastManager(SettingsManagerPointer pSettingsManager,
```

Comment:
Yes. Shard pointer for shared ownership. With a plain pointer parameter, the function promises not to store the pointer.